examples/js/controls/OrbitControls.js

```diff
@@ -1178,3 +1207,117 @@ THREE.MapControls = function ( object, domElement ) {
 
 THREE.MapControls.prototype = Object.create( THREE.EventDispatcher.prototype );
 THREE.MapControls.prototype.constructor = THREE.MapControls;
+
+// This set of controls performs orbiting, dollying (zooming), and panning.
+// Unlike OrbitControls, it rotates the "up" direction camera.up.
+//
+//    Orbit - left mouse / touch: one-finger move
+//    Zoom - middle mouse, or mousewheel / touch: two-finger spread or squish
+//    Pan - right mouse, or left mouse + ctrl/meta/shiftKey, or arrow keys / touch: two-finger move
+
+THREE.TrackballControls = function ( object, domElement ) {
```

Comment:
I really think this code and the definition of `MapControls` should go in its own module file. The following code looks weird, IMO:
```js
import { TrackballControls } from './jsm/controls/OrbitControls.js';
```

Yes, I'm aware that the JS versions would have a dependency to `OrbitControls` but I think this is still better than having strange imports.

Reply:
Hmm, not sure actually. This is more of a preference / library consistency kind of deal, so I think that's up to you guys. I'll be happy to change if we decide on that course of action.

Reply:
Sorry, I really don't understand your argumentation. We normally remove legacy/obsolete classes with very few exceptions. If we would keep all files, the repository would not be manageable anymore.

If users can't invest time in migration, it's okay to use one of the previous release. Hence, I don't get your goal with retaining the API of `TrackballControls`. It's good to align it without keeping the old stuff around.

Reply:
>If we would keep all files, the repository would not be manageable anymore.

Come on, dude.

Reply:
> I don't get your goal with retaining the API of TrackballControls. 

https://github.com/mrdoob/three.js/pull/17858#issuecomment-550055124